packages/docusaurus/src/commands/serve.ts

```diff
@@ -0,0 +1,54 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import http from 'http';
+import portFinder from 'portfinder';
+import serveHandler from 'serve-handler';
+import boxen from 'boxen';
+import chalk from 'chalk';
+import path from 'path';
+
+import build from './build';
+
+export default async function serve(
```

Comment:
We now use a `choosePort` fn copied from CRA for the docusaurus start command (instead of portfinder used before). I think we should use the same for docusaurus serve

Reply:
Fixed it. I already added docs.

Reply:
thanks, missed the doc ;)

can you have an option for --host too please? 

It's important in some cases to be able to specify it.

docusaurus start supports this too:
https://v2.docusaurus.io/docs/cli#docusaurus-start

Reply:
ok

Reply:
done